pandas/core/arrays/categorical.py

```diff
@@ -1492,6 +1530,16 @@ def __array__(self, dtype: NpDtype | None = None) -> np.ndarray:
             A numpy array of either the specified dtype or,
             if dtype==None (default), the same dtype as
             categorical.categories.dtype.
+
+        Examples
+        --------
+
+        >>> cat = pd.Categorical(['a', 'b'], ordered=True)
+
+        The following calls cat.__array__
```

Comment:
in general for code snippets we probably want double backticks:
```
The following calls ``cat.__array__``
```
(unless it's an argument name, in which case a single backtick is fine. the docs aren't terribly consistent about this admittedly)